data-access/src/object_store/mod.rs

```diff
@@ -91,6 +92,39 @@ pub trait ObjectStore: Sync + Send + Debug {
         })))
     }
 
+    /// Calls `list_file` with a glob_pattern
+    async fn glob_file(&self, path: &str) -> Result<FileMetaStream> {
```

Comment:
```suggestion
    async fn glob_files(&self, path: &str) -> Result<FileMetaStream> {
```

Should probably be plural for consistency